packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -122,7 +122,8 @@ export default class BuySellStore extends BaseStore {
         requestWS({
             p2p_advertiser_info: 1,
         }).then(response => {
-            if (!response.error) {
+            // Added a check to prevent console errors
+            if (response && !response.error) {
```

Comment:
This condition can just be:

```suggestion
            if (!response.error) {
```

Reply:
Why do we need to check for response?

Reply:
We can use optional chaining like `response?.error`

Reply:
To prevent console errors that arise if response is unavailable, I'll add a check to verify if the object is present, then evaluate the value of its properties.

![Screen Shot 2022-03-10 at 11 23 13](https://user-images.githubusercontent.com/98398322/157881342-01941142-3d45-4289-ae3d-dc8b2e552796.png)